packages/ckeditor5-engine/tests/manual/tickets/4600/1.js

```diff
@@ -19,6 +20,11 @@ import InputObserver from '../../../../src/view/observer/inputobserver';
 import KeyObserver from '../../../../src/view/observer/keyobserver';
 import MouseObserver from '../../../../src/view/observer/mouseobserver';
 import MouseEventsObserver from '@ckeditor/ckeditor5-table/src/tablemouse/mouseeventsobserver';
+import DeleteObserver from '@ckeditor/ckeditor5-typing/src/deleteobserver';
+import ClipboardObserver from '@ckeditor/ckeditor5-clipboard/src/clipboardobserver';
+import EnterObserver from '@ckeditor/ckeditor5-enter/src/enterobserver';
+import ImageLoadObserver from '@ckeditor/ckeditor5-image/src/image/imageloadobserver';
+import MutationObserver from '@ckeditor/ckeditor5-engine/src/view/observer/mutationobserver';
```

Comment:
Let's remove MutationObserver - it adds much noise and we're not really handling it for the time being.

Reply:
Done.